cmd/analyzer/parse_args.go

```diff
@@ -160,6 +169,8 @@ func ParseInArgs(cmdlineArgs []string) (*InArgs, error) {
 	args.VPC = flagset.String(VPC, "", "CRN of the VPC to analyze")
 	args.Debug = flagset.Bool(Debug, false, "Run in debug mode")
 	args.Version = flagset.Bool(Version, false, "Prints the release version number")
+	args.ConnectionDescription = flagset.String(ConnectionDescription, "",
+		"Description to a connection in the format: 'Protocol', 'MinSrc', 'MaxSrc'")
```

Comment:
Again, why is this required?

Reply:
though it will be easier for the user to pass single input for the connection description instead of each param (and I can parse it)

Reply:
I see. Perhaps I mislead you. Its better, I think, to use the existing mechanism for feeding parms. This why we can use the existing parsing and validity mechanism. E.g,use the go args mechanism to enforce that ports parm are ints withing range